packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -2272,6 +2282,7 @@ export default class Meeting extends StatelessWebexPlugin {
     this.locusInfo.on(EVENTS.LOCUS_INFO_UPDATE_URL, (payload) => {
       this.members.locusUrlUpdate(payload);
       this.breakouts.locusUrlUpdate(payload);
+      this.annotation.locusUrlUpdate(payload);
```

Comment:
issue: There should be tests to check that the locus url and approval url update are called

Reply:
Done